install.sh

```diff
@@ -42,11 +69,51 @@ STEAM_UID=$(grep "^${STEAM_USER}" /etc/passwd | cut -d':' -f3)
 STEAM_GID=$(grep "^${STEAM_USER}" /etc/passwd | cut -d':' -f4)
 echo "Steam user '${STEAM_USER}' found with UID ${STEAM_UID} and GID ${STEAM_GID}"
 
+# Choosing from the guide in Proton (SteamPlay) Wiki https://github.com/ValveSoftware/Proton/wiki/Requirements
+# Check Graphic card
+echo "What is your Graphic Card Manufacturer"
+echo "1) Nvidia"
+echo "2) AMD"
+echo "3) Intel"
+read case;
+
+case $case in
+	1)	echo "Getting Latest Graphic Drivers..."
+		add-apt-repository ppa:graphics-drivers/ppa
+		apt update
+		apt install nvidia-driver-415 -y;;
+	2)	echo "Getting Latest AMD Graphic Drivers..."
+		add-apt-repository ppa:oibaf/graphics-drivers
+		apt update
+		apt apt -y upgrade;;
+	3)	echo "Getting Latest Intel Graphic Drivers..."
+		add-apt-repository ppa:paulo-miguel-dias/pkppa
+		apt update
+		apt dist-upgrade
+		apt install mesa-vulkan-drivers mesa-vulkan-drivers:i386 -y;;
+esac 
+
 # Install steam and steam device support.
 echo "Installing steam..."
 apt update
 apt install steam steam-devices -y
 
+# WIP - find a way to enable Steamplay without using Desktop Steam Client. Also maybe find a way to enable Steam Beta with latest Steamplay
```

Comment:
It looks like Steam Play can be enabled through the `~/.steam/config/config.vdf` configuration file:
![image](https://user-images.githubusercontent.com/376460/50619203-af511080-0eac-11e9-950e-28810e915eef.png)

I found this result by comparing the config before and after enabling Steam Play.

Reply:
Activating the Proton beta at start is awesome, but this file doesn't exist on my setup (regular Ubuntu, not steamos-ubuntu). It's rather `~/.steam/steam/config/config.vdf`
As for the config part, here is mine:
```
                                "Tools"
                                {
                                        "enable"                "1"
                                }
                                "ToolMapping"
                                {
                                        "0"
                                        {
                                                "name"          "proton_37"
                                                "config"                ""
                                                "Priority"              "75"
                                        }
                                }
```

Reply:
The script could look in both locations for the configuration. There's a Python library for parsing Valve's VDF format that could be used to load the current configuration, update the values, and write it back out:
https://pypi.org/project/vdf/